lib/fittings/error_handler.js

```diff
@@ -23,9 +25,15 @@ module.exports = function create() {
                 res.locals.errorStatus = err.status;
             }
 
-            var stackDump = err.stack.replace(/\n/g, '');
             message = message.replace(/\n/g, '');
-            res.body = {message: message, status: res.locals.errorStatus, stack: stackDump};
+            res.body = {message: message, status: res.locals.errorStatus};
+            var minLogLevel = nconf.get('minLogLevel');
+            if (minLogLevel === constants.Logging.Levels.debug) {
+                if (err.stack !== undefined) {
+                    var stackDump = err.stack.replace(/\n/g, '');
```

Comment:
Second on this.

Below is an example of error output:
```
{
    "message": "Could not find node with identifier &#34;57a7fe2dd84b11337a3293a6&#34;",
    "stack": "NotFoundError: Could not find node with identifier &#34;57a7fe2dd84b11337a3293a6&#34;    at /home/yuanf/src/on-core/lib/common/model.js:292:27    at bound (domain.js:287:14)    at runBound (domain.js:300:12)    at tryCatcher (/home/yuanf/src/on-core/node_modules/waterline/node_modules/bluebird/js/main/util.js:26:23)    at Promise._settlePromiseFromHandler (/home/yuanf/src/on-core/node_modules/waterline/node_modules/bluebird/js/main/promise.js:503:31)    at Promise._settlePromiseAt (/home/yuanf/src/on-core/node_modules/waterline/node_modules/bluebird/js/main/promise.js:577:18)    at Async._drainQueue (/home/yuanf/src/on-core/node_modules/waterline/node_modules/bluebird/js/main/async.js:128:12)    at Async._drainQueues (/home/yuanf/src/on-core/node_modules/waterline/node_modules/bluebird/js/main/async.js:133:10)    at Immediate.Async.drainQueues [as _onImmediate] (/home/yuanf/src/on-core/node_modules/waterline/node_modules/bluebird/js/main/async.js:15:14)    at processImmediate [as _immediateCallback] (timers.js:383:17)",
    "status": "404"
}
```

I don't think the stack is the information that should export to user even in debug level if we don't convert it to be more user-friendly.  Also, we'd better to decode the message and stack string to remove  those characters like "&#34;"